### ISSUE TITLE
SDK crashes app when choosing View License on Activate License dialog 

### ISSUE THREAD

**Issue by gmoens-vuzix** [webex/webex-android-sdk#32]
Using SDK 2.6.0.

The dialog that pops up asking you to activate a license, if you choose View License, the app crashes with the following stack trace:

android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
        at android.app.ContextImpl.startActivity(ContextImpl.java:912)
        at android.app.ContextImpl.startActivity(ContextImpl.java:888)
        at android.content.ContextWrapper.startActivity(ContextWrapper.java:379)
        at com.ciscowebex.androidsdk.phone.internal.PhoneImpl.lambda$dial$15$PhoneImpl(PhoneImpl.java:336)

**Comment by olivierproffit** [comment]
Thanks @gmoens-vuzix for reporting. We are working on it as we speak.

**Comment by kliu** [comment]
Please refresh the gradle dependencies for the fix.

**Comment by gmoens-vuzix** [comment]
Confirmed. I had to delete 2.6.0 from my gradle cache. But once I did, it downloaded it again and the fix was there. Thanks!

**Comment by gmoens-vuzix** [comment]
This error is back again, same stack trace. Clearing gradle cache did not help.

**Comment by gmoens-vuzix** [comment]
When I download the zip file (webex-android-sdk-2.6.0.zip) and look at the PhoneImpl.java source, I see the fix is there:

                Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(H264LicensePrompter.LICENSE_URL));
                browserIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                this.getContext().startActivity(browserIntent);

But the artifacts downloaded via gradle/maven (androidsdk-2.6.0-sources.jar), no longer have the fix:

                Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(H264LicensePrompter.LICENSE_URL));
                this.getContext().startActivity(browserIntent);

**Comment by KunNiu** [comment]
@gmoens-vuzix , you can try the snapshot version 2.7.0-SNAPSHOT as follow steps
1, Add snapshot maven repo to project's build.gradle

```
maven {
            url 'https://devhub.cisco.com/artifactory/webexsdk-SNAPSHOT/'
        }
```
2, Change version number in app's build.gradle

```
implementation('com.ciscowebex:androidsdk:2.7.0-SNAPSHOT@aar', {
        transitive = true
        changing = true
    })
```

**Comment by gmoens-vuzix** [comment]
@KunNiu yes, 2.7.0-SNAPSHOT works fine. This appears to only be a problem with the 2.6.0 artifacts that come from artifactory. Back on Oct 1st, those were working.

**Comment by gmoens-vuzix** [comment]
This is resolved with the release of SDK 2.7.0.
